pandas/tests/reshape/test_pivot.py

```diff
@@ -1724,12 +1723,12 @@ def test_daily(self, i):
 
         doy = np.asarray(ts.index.dayofyear)
 
-        subset = ts[doy == i]
```

Comment:
Honestly, it may be a better test to use `tm.assert_frame_equal` and compare this to an expected DataFrame (making the input have constant data)

Reply:
yeah, that makes sense. updated